docs/checklist.md

```diff
@@ -27,6 +27,48 @@ Comes in three sections. You need varying degrees of access depending on where y
 - ChecklistProvider: Wraps around the carousel so that the toggle buttons in the secondary menu know when the checklist is open and the other popup menus there (help center) can respect it. 
 - ChecklistCountProvider, ChecklistCategoryProvider: Wrap around various parts of the checklist itself to provide counts of issues.
 
+## Roles
+
+There are six possible roles that a user may be:
+
+- Super Admin
+- Administrator
+- Editor
+- Author
+- Contributor
+- Subscriber
+
+Depending on the role, the user may or may not be able to fix specific checklist items. Due to this constraint, some of the checklist items will be hidden based on the user's role.
+
+**NOTE**: `Contributor`s are only able to edit _non-published_ stories. Once a story is published, many cards will be hidden from `contributor`s as well.
+
+This is outlined in the table below:
+
+|Story issue|Users that will be shown the checklist card|Users that will be shown the checklist card when the story has been published|
```

Comment:
Curious about the distinction between published vs non-published. What's the rationale here?

Reply:
It's my understanding that once a story is published it can only be edited by users with the `edit_published_web-stories` permission. Is that correct?

After reading your comment above though this column is not necessary since only users that are able to edit the story are able to see the checklist